DataFormats/GEMDigi/src/classes_def.xml

```diff
@@ -46,4 +46,28 @@
 <class name="MuonDigiCollection<ME0DetId,int>"/> 
 <class name="edm::Wrapper<MuonDigiCollection<ME0DetId,int> >" splitLevel="0"/> 
 
+<class name="ME0Digi" ClassVersion="10">
+ <version ClassVersion="10" checksum="2570759453"/>
+</class>
+<class name="std::vector<ME0Digi>"/>
+<class name="std::map<ME0DetId,std::vector<ME0Digi> >"/>
```

Comment:
Because `classes.h` defines `std::vector<std::vector<ME0Digi> >  vvmm;`, no reference to maps.

Also, serializing `std::map` may still be inadvisable.